orb/core/model.py

```diff
@@ -108,7 +108,8 @@ def _allowed(columns=None, context=None):
                     not auth(columns=(column,), context=context)):
                 continue
 
-            elif column.testFlag(column.Flags.Virtual) and not isinstance(self, orb.View):
+            elif ((column.testFlag(column.Flags.Virtual) and not isinstance(self, orb.View)) or
+                   column.gettermethod() is not None):
```

Comment:
gettermethods provide overrides to the standard logic, so they should always be called.  not all columns will provide a gettermethod override anymore